src/win32/toga_win32/app.py

```diff
@@ -46,3 +46,6 @@ def on_set_focus(self, msg, old_focus, wParam):
     def on_activate(self, msg, state, lParam):
         if state == WA_INACTIVE:
```

Comment:
At column 21: (F405) 'WA_INACTIVE' may be undefined, or defined from star imports: .libs